ts/IterableInboxEmptyState.tsx

```diff
@@ -40,7 +43,7 @@ const styles = StyleSheet.create({
       paddingBottom: 25
    },
 
-   subtitle: {
+   body: {
       fontSize: 15,
       color: 'grey'
```

Comment:
Wont this [body: fontSize] override with customizationType.ts -> [body: fontSize] ?

Reply:
Oh got it. It rather passes it as argument to it.